packages/web/config/feature-flag.ts

```diff
@@ -65,3 +65,8 @@ export const PromotedLBPPoolIds: {
       },*/
     ]
   : [];
+
+export const ICNSInfo = {
```

Comment:
May be helpful to link this as well

https://www.mintscan.io/osmosis/wasm/contract/osmo1xk0s8xgktn9x5vwcgtjdxqzadg88fgn33p8u9cnpdxwemvxscvast52cdd